index.html

```diff
@@ -30,42 +30,41 @@ <h2></h2>
       <p>Ice cream: In a small pot over medium heat, combine milk, and granulated sugar until sugar is completely disolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled.</p>
       <p>Using an ice cream machine, pour liquids into the frozen bassin and process according to manufacturer instruction, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture and the nuts. Continue process for +/- 10 minutes. Pour semi.frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving.</p>
 
     <form>
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">
-      <input type="checkbox">  
+        <input type="checkbox" id="milk"> <label for="milk">1 1/2 cups milk</label>
+        <input type="checkbox" id="mascarpone"> <label for="mascarpone">1/2 cup mascarpone</label>
+        <input type="checkbox" id="pink"> <label for="pink">1/2 tsp pink salt</label>
+        <input type="checkbox" id="figs"> <label for="figs">1 lb Black Mission Figs</label>
+        <input type="checkbox" id="bro_sugar"> <label for="bro_sugar">1/ cup brown sugar</label>
+        <input type="checkbox" id="water"> <label for="water">2-4 tbs water</label>
+        <input type="checkbox" id="cream"> <label for="cream">1 1/2 cups heavy cream</label>
+        <input type="checkbox" id="gra_sugar"> <label for="gra_sugar">4/3 granulated sugar</label>
+        <input type="checkbox" id="yolks"> <label for="yolks">2 egg yolks</label>
+        <input type="checkbox" id="lemon"> <label for="lemon">1 lemon juiced</label>
+        <input type="checkbox" id="butter"> <label for="butter">2 tbsp butter</label>
+        <input type="checkbox" id="honey"> <label for="honey">1 cup honey roasted pecans, roughly chopped</label>   
     </form>
     <hr>
   </main>
 
   <footer>
     <div>
-      <img>
-      <h3></h3>
-      <p></p>
+      <img src="img/van-pic.png">
+      <h3>Vanessa Stevenson</h3>
+      <p>Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much whi I am.</p>
     <form>
-      <input>
+      <input type="button" id="share"><label for="share">SHARE RECIPIE</label>
     </form>
     </div>
-    <img>
-    <p></p>
+    <img src="img/small-logo.png">
+    <p>Delicious © 2013 · All Rights Reserved.</p>
+    <P>Proudly published with Ghost.</P>
   </footer>
-
-
 </body>
 
 </html>
```

Comment:
Style nitpick: GitHub likes there to be one blank line at the end of every file, which is why you get this angry red symbol here.